apps/cookbook/src/app/examples/dropdown-example/dropdown-example.component.html

```diff
@@ -8,6 +8,9 @@ <h3>Simple</h3>
 <cookbook-dropdown-example-pre-selected [size]="size"></cookbook-dropdown-example-pre-selected>
 <cookbook-dropdown-example-expand [size]="size"></cookbook-dropdown-example-expand>
 <cookbook-dropdown-example-right-aligned [size]="size"></cookbook-dropdown-example-right-aligned>
+<cookbook-dropdown-example-dropdown-right-aligned
+  [size]="size"
+></cookbook-dropdown-example-dropdown-right-aligned>
```

Comment:
Consider if this example could be merged with existing right-aligned example (showcasing right alignment + open to the left in one example)

Reply:
I'll do that. The reason I opted not to, was that the right aligned example triggers the interception observer and is as such demo-ing another implementation detail. But I guess that is kind of irrelevant for the cookbook consumer

Reply:
Valid point 👍 
But on larger screens (ie. room for default opening direction) we get this 👇  which isn't demoing the right alignment every well anyhoo 🙈 
![image](https://user-images.githubusercontent.com/9556693/107010596-d585c080-6796-11eb-9679-305e226c8830.png)

Reply:
Done